### PR TITLE
Email

### PR DESCRIPTION
**Pull request by TimoA200**
Why does Mongoose needs so much time to connect?
Btw, have fun lmao

### CODE REVIEW

File: vobe.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "version": "0.0.2-alpha"
-}
\ No newline at end of file
+  "version": "0.0.3-alpha",
+  "authMail": "auth@vobe.io",
+  "mailUsername": "auth@vobe.io",
+  "mailPassword": "Haha, did you really think we would leak our password? ¯\_(ツ)_/¯"
```
Comment (Comeza) [inline vobe.json]: maybe we should move secrets to an entire new file.

File: bin/server.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+import MailModule from "./modules/MailModule.js";
 import Table from 'cli-table3';
 import * as fs from 'fs';
 import colors from 'colors';
 import MongooseModule from "./modules/MongooseModule.js";
 
 colors.enabled = true;
 
-let vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
+global.vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
```
Comment (Comeza) [inline bin/server.js:11]: Maybe we could move the vobeJson to an entire new file and export it from there.
So you only have to use
```js
import vobeJson from 'vobeJson.js'
```

Using global variables is fine at the moment, but can make things worse later.

File: bin/gql/resolver/Query/signup.js [hunk 3]
```diff
@@ -41,5 +43,16 @@ export default async (parent, args, context, info) => {
...
 
+    new VerificationModel({
+        email: email
+    }).save();
+
+    await transporter.sendMail({
+        from: '"Vobe" <auth@vobe.io>',
+        to: email,
+        subject: "Please verify your account",
+        text: "Hey, you've just created an account on vobe.io. Please click on the link below to verify it. https://beta.vobe.io/verification/32093092",
```
Comment (Comeza) [inline bin/gql/resolver/Query/signup.js]: I don't think it's right that there is a fixed verification link in there.

Comment (TimoA200) [reply bin/gql/resolver/Query/signup.js]: You're right, but the the verification feature isn't implemented in vobe-web so far

File: bin/server.js (hunk 2 above)
Comment (TimoA200) [reply bin/server.js:11]: Do you mean that the variables are stored directly in the vobeJson.js files as follows?
```js
const variable1 = something;
const variable2 = something;
const variable3 = something;
```
Or do you mean that the vobeJson.js parses the vobe.json and saves it in a vobeJson variable like currently in server.js
```js
const vobeJson = JSON.parse(fs.readFileSync('vobe.json').toString());
```

File: vobe.json (hunk 1 above)
Comment (TimoA200) [reply vobe.json]: You're right, but maybe not in this branch.

File: bin/gql/resolver/Query/signup.js (hunk 3 above)
Comment (TimoA200) [reply bin/gql/resolver/Query/signup.js]: Are you okay with the latest change?

**Pull request merged**